lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -177,6 +186,101 @@ def __init__(self, *args):
 The consumer charm can then choose to update its configuration in both situations.
 
 
+## LogProxyConsumer Library Usage
+
+Let's say that we have a workload charm that produces logs and we need to send those logs to a
+workload implementing the `loki_push_api` interface, such as `Loki` or `Grafana Agent`.
+
+Adopting this object in a charmed operator consist of two steps:
+
+
+1. Use the `LogProxyConsumer` class by instanting it in the `__init__` method of the charmed
+   operator. There are two ways to get logs in to promtail. You can give it a list of files to read
+   or you can write to it using the syslog protocol.
+
+   For example:
+
+   ```python
+   from charms.loki_k8s.v0.log_proxy import LogProxyConsumer, PromtailDigestError
+
+   ...
+
+       def __init__(self, *args):
+           ...
+           try:
+               self._log_proxy = LogProxyConsumer(
+                   charm=self, log_files=LOG_FILES, container_name=PEER, syslog=True
```

Comment:
I think there may have been a bit of a merge issue here. This should say `enable_syslog=True`